src/crypto/public_key.h

```diff
@@ -14,6 +14,12 @@
 
 namespace crypto
 {
+  static inline std::string get_public_key_pem_hash_hex(
```

Comment:
Can you have a look at https://github.com/microsoft/CCF/issues/1378#issuecomment-771581216? It seems like this is taking the hash over the PEM and not DER.

Reply:
This is now done: we use the SHA-256 hash of the DER-encoded public key (hex-encoded) as the node ID (this seems to be standard, e.g. https://stackoverflow.com/questions/40404963/how-do-i-get-public-key-hash-for-ssl-pinning). We also use the same value in the SGX quote's report data.